user/build-stages.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Build Stages
+permalink: /user/build-stages/
+layout: en
+---
+
+<div id="toc"></div>
+
+> Build stages are currently in BETA. There is more information about what this means, and how you can give us feedback on this new feature in the [GitHub issue](https://github.com/travis-ci/beta-features/issues/11).
+{: .beta}
+
+With this new feature you can group jobs together in 'stages'. Jobs in a stage
+run in parallel, and the stages themselves run sequentially, one after another.
+The build fails when any stage fails, i.e. if any job in this stage fails.
+
+For example, you can configure a deployment stage to run only after several test
+jobs have all completed successfully.
+
+Assign jobs to stages by adding a stage name to the job configuration
+in the `jobs.include` section of your `.travis.yml` file:
+
+```yaml
+jobs:
+  include:
+    - stage: test
+      script: ./test 1
+    - stage: test
+      script: ./test 2
+    - stage: deploy
+      script: ./deploy
+```
+
+The previous yaml creates a build with three jobs, two of which start in
+parallel in the first stage (named `test`), while the third job on the second
+stage (named `deploy`) starts only after the test stage completes successfully.
+
+This screencast demonstrates how the two stages work:
+
+![Example screencast](https://cloud.githubusercontent.com/assets/3729517/25229553/0868909c-25d1-11e7-9263-b076fdef9288.gif)
+
+## Naming your stages
+
+Stage names are arbitrary strings, and can include spaces or emojis. The first
+letter of a stage name is automatically capitalized for aesthetical reasons, so
+you don't have to deal with uppercase strings in your `.travis.yml` file.
+
+The default stage is `test`. Jobs that do not have a stage name are assigned to
+the previous stage name if one exists, or the default stage name if there is no
+previous stage name. `test`. This means that if you set the stage name on the
```

Comment:
Not sure if the `test` mention in this line is a typo (is it?), since we already define the default stage name two lines above.